moosefs_tricorder/common.py

```diff
@@ -23,17 +23,19 @@ def run(command: str):
     return (output, err)
 
 
-def load_chunkserver_metrics(moosefs_master: str) -> dict:
+def load_chunkserver_metrics(moosefs_master: str, moosefs_master_port: int) -> dict:
```

Comment:
I like that you've made the port configurable. Could you set the default value of `moosefs_master_port` to 9420? that's the default moosefs master port, iirc.

Reply:
i dont understand this. 

The default master port is set at [arg_parse](https://github.com/unixorn/prometheus-moosefs-tricorder/blob/main/moosefs_tricorder/cli.py#L30). 
This value is [passed to MooseCollector contructor initialize](https://github.com/unixorn/prometheus-moosefs-tricorder/blob/main/moosefs_tricorder/moosefs.py#L302) , [constructor "save"](https://github.com/unixorn/prometheus-moosefs-tricorder/blob/main/moosefs_tricorder/moosefs.py#L33) the value. And load_*_metrics functions are called with setted master_port.

Reply:
I was only looking at the PR, my mistake.

Thanks for the contribution!